rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -1085,56 +1090,53 @@ where
     }
 
     /// Remove the SNS wasm canister as the controller of the canisters
-    async fn remove_self_as_controller(
+    async fn remove_sns_w_as_controller(
         canister_api: &impl CanisterApi,
         canisters: &SnsCanisterIds,
     ) -> Result<(), String> {
         let set_controllers_results = vec![
-            // Removing self, leaving SNS Root.
+            // Removing SNS-W, leaving SNS Root.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.governance.unwrap()),
                     vec![canisters.root.unwrap()],
                 )
                 .await
-                .map_err(|e| {
-                    format!(
-                        "Unable to remove SNS-WASM as Governance's controller: {}",
-                        e
-                    )
+                .map_err(|err| {
+                    format!("Unable to remove SNS-W as Governance's controller: {}", err)
                 }),
-            // Removing self, leaving SNS Root.
+            // Removing SNS-W, leaving SNS Root.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.ledger.unwrap()),
                     vec![canisters.root.unwrap()],
                 )
                 .await
-                .map_err(|e| format!("Unable to remove SNS-WASM as Ledger's controller: {}", e)),
-            // Removing self, leaving SNS Governance.
+                .map_err(|err| format!("Unable to remove SNS-W as Ledger's controller: {}", err)),
+            // Removing SNS-W, leaving SNS Governance.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.root.unwrap()),
                     vec![canisters.governance.unwrap()],
                 )
                 .await
-                .map_err(|e| format!("Unable to remove SNS-WASM as Root's controller: {}", e)),
-            // Removing self, leaving SNS Root and NNS Root.
+                .map_err(|err| format!("Unable to remove SNS-W as Root's controller: {}", err)),
+            // Removing SNS-W, leaving SNS Root and NNS Root.
             canister_api
                 .set_controllers(
                     CanisterId::unchecked_from_principal(canisters.swap.unwrap()),
-                    vec![canisters.root.unwrap(), ROOT_CANISTER_ID.get()],
+                    vec![canisters.root.unwrap()],
```

Comment:
I already made this point earlier, but I want to reiterate, because I think this is important:

It is hard to see the implementation of **_actual behavior changes_**, because there are so many OTHER changes.

It is difficult to detect a signal when there is a whole bunch of noise around.

Reply:
Agree in general. Note that this would not have been a problem were the tests in their own file. 

For now, let's keep it since you reviewed the code anyway.